src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  margin: auto;
+  padding: 10px 50px;
```

```suggestion
  padding: 0 50px;
```
Не впевнений, що вертикальні падінги тут порібно